frontend/src/ches/MergeForm.jsx

```diff
@@ -11,6 +11,7 @@ import ChesValidationError from './ChesValidationError';
 import ChesAlertList from './ChesAlertList';
 import AlertDisplay from '../utils/AlertDisplay';
 import GetUserError from '../auth/GetUserError';
+import moment from 'moment';
```

Comment:
`MergeForm.jsx` is more than 1k lines long - may be worth looking to see if there are any other ways to break down this React component into smaller pieces to make it easier to maintain in the future.

Reply:
refactoring later when we add status screen for CHES